### ISSUE TITLE
Question about data overlap

### ISSUE THREAD

**Issue by lichengshen** [zamling/PSALM#16]
Hello, thanks for the great work.

I noticed that the RefCOCO val/test sets use images from the COCO training set. When doing joint training, I think this could cause a data leak, that the testing images and masks for RefCOCO are seen when training on COCO-Panoptic. Is this true, or have you handled this somewhere?

**Comment by Z-MU-Z** [comment]
Hello, I noticed that LISA mentioned in the paper 'we exclude the COCO samples whose images are present in the refCOCO(+/g) validation sets during training.' However, it seems that I didn't find this implemented in the code. Did you notice anything regarding this?"

**Comment by zamling** [comment]
Hi, all

I did not notice this problem. We built our dataset based on [LISA](https://github.com/dvlab-research/LISA) and [UNINEXT](https://github.com/MasterBin-IIAU/UNINEXT/blob/master/assets/DATA.md). RefCOCO/+/g and COCO train2017 are built from LISA, so I do not know such data leak has been dealed well. So can you provide me some codes in [LISA](https://github.com/dvlab-research/LISA) or [UNINEXT](https://github.com/MasterBin-IIAU/UNINEXT/blob/master/assets/DATA.md) about this processing? So that I can check wheter I follow it correctly.
